src/mmrelay/constants/queue.py

```
@@ -5,16 +5,62 @@
+    "MAX_QUEUE_SIZE",
+    "QUEUE_HIGH_WATER_MARK",
+    "QUEUE_MEDIUM_WATER_MARK",
+    "QUEUE_LOG_THRESHOLD",
+    "TASK_SHUTDOWN_TIMEOUT_SEC",
+    "QUEUE_FULL_LOG_INTERVAL_SEC",
+    "QUEUE_WAIT_RETRY_SLEEP_SEC",
+    "QUEUE_POLL_INTERVAL_SEC",
+    "CONNECTION_RETRY_SLEEP_SEC",
+    "QUEUE_EXECUTOR_MAX_WORKERS",
+    "CONNECTION_ERROR_KEYWORDS",
+]
```

Ruff (0.15.7)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)